modul/i18n.py

```
@@ -0,0 +1,48 @@
+
+    This consults the runtime `current_settings.language` when available.
+    """
+    try:
+        from modul import settings as settings_mod
+
+        lang = getattr(settings_mod, "current_settings", None)
+        if lang and getattr(lang, "language", None):
+            return t(key, lang.language)
+    except Exception:  # pylint: disable=broad-exception-caught
+        # If settings cannot be inspected, fall back to English
+        pass
```

Ruff (0.14.13)

`try`-`except`-`pass` detected, consider logging the exception

(S110)